README.md

```diff
@@ -17,6 +17,15 @@ You'll need [Rust](https://www.rust-lang.org/tools/install) to build the parser
 cargo run <PATH_TO_FILE>
 ```
 
+## Running the UI
+To show the chart in the browser, you can start development server:
+```
+cd heapview/src/ui/
+npm install
+npm start
```

Comment:
```suggestion
yarn install
yarn start
```

Reply:
Hi! Thanks for suggestion! I've noticed that yarn.lock contains urls to the artifactory. It would be great if you could open new PR with change you suggested and removal (or replacement) of the lock file?